src/Raven.Server/Web/System/DatabasesHandler.cs

```diff
@@ -346,8 +345,8 @@ await using (var writer = new AsyncBlittableJsonTextWriter(context, ResponseBody
                              dbTask != null &&
                              dbTask.IsCompleted;
 
-                var dbRecord = JsonDeserializationCluster.DatabaseRecord(dbRecordBlittable);
-                var topology = dbRecord.Topology;
+                var dbRecord = rawDatabaseRecord.MaterializedRecord;
```

Comment:
Can we avoid materializing the whole record here? Otherwise, why get the `RawDatabaseRecord` and not the `DatabaseRecord` in the first place?

Reply:
I think we do not have to materialize whole record here?

Reply:
By not materializing the whole record you mean extracting only the needed parts from the blittable? Instead of using `JsonDeserializationServer`?
`MaterializedRecord` does the exact same thing as line 349 plus it handles the disposing of the blittable so I thought it would be better to reuse that code while keeping the same behavior

Reply:
Done